examples/shader/custom_post_processing.rs

```diff
@@ -365,9 +366,9 @@ fn rotate(time: Res<Time>, mut query: Query<&mut Transform, With<Rotates>>) {
 // Change the intensity over time to show that the effect is controlled from the main world
 fn update_settings(mut settings: Query<&mut PostProcessSettings>, time: Res<Time>) {
     for mut setting in &mut settings {
-        let mut intensity = time.elapsed_seconds().sin();
+        let mut intensity = sin(time.elapsed_seconds());
         // Make it loop periodically
-        intensity = intensity.sin();
+        intensity = sin(intensity);
```

Comment:
Wtf is this code lol. Two sines in a row is a sign something is off, and the comment doesn't fill me with confidence either.

I suppose it's not a fault of this PR though.